docs/2_2_common_mechanisms.adoc

```diff
@@ -330,522 +343,711 @@ Set <<parameter,`parameters`>>, <<input,`inputs`>>, and <<start>> values, and re
 
-_[TODO: shouldn't we use constant and fixed like for other variable variabilities (strictly periodic = constant periodic; periodic = fixed periodic; aperiodic = tunable periodic! TorstenBlochwitz will start an issue to collect use cases to clarify clock use.)_
-_Does the importer have to call fmi3SetInterval for aperiodic clocks at each clock tick if the clock should tick with the same interval? Or will there be only one clock tick per fmi3SetInterval call?
-FMI Design meeting: remark that set interval functions are informative to design model based controllers.
-The environment is the one that will tick.
-]_
+===== Model Partitions and Clocked Variables [[clocked-variable]]
 
-A <<clock>> is <<strict,strictly>> <<periodic>> if latexmath:[\Delta T] is fixed for an FMU and cannot be changed.
-The <<clock>> is periodic, if latexmath:[\Delta T] can be defined before calling `fmi3EnterInitializationMode` via <<fmi3SetIntervalDecimal>> or <<fmi3SetIntervalFraction>>.
-The <<clock>> is <<periodic,aperiodic>> if latexmath:[\Delta T] is not constant during a simulation run (<<fmi3SetIntervalDecimal>> or <<fmi3SetIntervalFraction>> can be called for that <<clock>> during *Event Mode*).
+A <<clock>> activates its specific model partition.
 
-===== Connecting Clocked FMUs [[connecting-clocked-fmus]]
+_[Such a model partition may represent a task or an interrupt service routine of an embedded system, or a discretized part of a plant model.]_
 
-When connecting imported FMUs, <<input>> and <<output>> variables can readily be connected only when they are defined on a <<clock>> with identical properties.
-If this is not possible, an explicit cast of one <<clock>> to another <<clock>> is typically defined in the environment.
+Discrete-time variables computed in such a model partition are called <<clocked-variable,clocked variables>>.
+Therefore, the value of a <<clocked-variable,clocked variable>> changes only when its clock is active.
+Model partitions are initialized at their first clock activation, not in <<InitializationMode>>.
 
-_[TODO: What is a clock cast? If a clock cast can be done by the environment anyway, why mention which inputs/outputs can be connected?_
-_Remove the entire section: we do not want to say: don't be stupid.]_
+In some clock semantics (e.g. <<MLS12>>), a clocked variable has a value only when its associated clock is active.
+However, `fmi3Get{VariableType}` will return the value computed at the last clock activation time.
+Before the first clock tick, clocked variables have their initial value.
 
-The environment can evaluate required <<clock>> casts for connections using the information on the <<clockReference>> attribute of assigned variables _[i.e. the variables that are assigned to a <<clock>> based on <<modelDescription.xml>> information]_.
+The association between <<clocked-variable,clocked variables>> and their <<clock,clocks>> is defined by the <<model-dependencies,model dependencies>>.
```

Comment:
Could we explain this more explicitely, please?
My understanding when reading this: 
We have Ouputs or DiscreteTimeStates listed in the ModelStructure, which have in their dependendency list at least one clock, so they are clocked. But what about other (local) variables, that are not listed in the modelStructure and shall be clocked?

Reply:
TODOs in the document and another ticket (#1287) address this.